### DISCUSSION TITLE
Is there a way to test the amp circuits

### DISCUSSION THREAD

**Discussion by RyanParsley** [cardonabits/haxo-hw#29] [General]
I'm concerned about my surface mount soldering. While testing the diodes with an multimeter was easy enough, I'm not so confident in how to confirm the amp is soldered well without simply plugging it in and hoping for the best. Is there a sensible way to test this that I'm not aware of?

**Comment by jcard0na** [comment]
Hi Ryan,

"plugging it and hoping for the best" was the least time-consuming approach that I found.  Fortunately the PCB assembly house I used did a very good job: not a single IC was faulty.

Cheers,
